jobs/nessus-manager/templates/config/crontab

```diff
@@ -1,4 +1,5 @@
 # m h dom mon dow user command
 # purge all our agents before our daily scans run, the agents will relink themselves at the bottom of the hour
-20 0 * * * root /var/vcap/jobs/nessus-manager/bin/purge-agents.sh
+20 0 * * * root /var/vcap/jobs/nessus-manager/bin/purge-agents.sh unlink
+55 0 * * * root /var/vcap/jobs/nessus-manager/bin/purge-agents.sh delete
```

Comment:
since we're now filtering for offline agents - any reason not to run this every hour?

Reply:
Good idea. And I'm changing the sequence to unlink at :10, delete at :20, agents relink at :30.  This matches what I've done manually of late.